bool deltaOccupancyGridRequired
---
comp0037_mapper/MapUpdate initialMapUpdate